.travis.yml

```diff
@@ -1,9 +1,10 @@
 # whitelist
 branches:
   only:
-      - u/fjammes/DM-1146 
+      - master
+      - tickets/DM-5319
 
-sudo: required
+sudo: false
```

Comment:
Yes, doc recommends using `required`: [Travis/Docker doc](https://docs.travis-ci.com/user/docker/), but this prevents using caching...